webapp/channels/src/components/analytics/true_up_review.tsx

```diff
@@ -223,10 +223,6 @@ const TrueUpReview: React.FC = () => {
         return null;
     }
 
-    if (telemetryEnabled) {
-        return null;
-    }
-
```

Comment:
If telemetry is enabled we still don't want to show the button to submit the True up review correct? When telemetry is enabled we already have the data that we need on our side for true up verifications

Reply:
Ah, okay. So should this be a recurring check on a schedule in the backend to just automatically send the true-up data off to CWS?

Reply:
Ah, my mistake. I thought this was negated. For some reason, I had thought that the feature was only enabled when telemetry is enabled.

Reply:
> So should this be a recurring check on a schedule in the backend to just automatically send the true-up data off to CWS?

We still want the button so they explicitly share the data with us but we just need to route it through the portal given that telemetry is disabled in their server